view/frontend/web/js/view/delivery-options.js

```diff
@@ -185,11 +187,18 @@ define(
        * @param {Object?} address - Quote.shippingAddress from Magento.
        */
       updateAddress: function(address) {
+        var newAddress;
+
         if (!deliveryOptions.isUsingMyParcelMethod) {
           return;
         }
 
-        window.MyParcelConfig.address = deliveryOptions.getAddress(address || quote.shippingAddress());
+        newAddress = deliveryOptions.getAddress(address || quote.shippingAddress());
+        if (JSON.stringify(newAddress) === JSON.stringify(window.MyParcelConfig.address)) {
```

Comment:
Wees wel bewust van het feit dat dit niet werkt als de volgorde van de properties verschilt. Als dat niet mogelijk is dan is deze oplossing prima, maar dit kan zo dus wel gekke bugs veroorzaken.

Reply:
Ik denk dat dit fout gaat zodra je een postcode checker heb waarbij je alleen de postcode + huisnummer hoeft in te vullen. Hierbij zal vaak de volgorde veranderen.